install-slsa-provenance.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+shopt -s expand_aliases
+if [ -z "$NO_COLOR" ]; then
+  alias log_info="echo -e \"\033[1;32mINFO\033[0m:\""
+  alias log_error="echo -e \"\033[1;31mERROR\033[0m:\""
+  alias log_warning="echo -e \"\033[1;33mWARN\033[0m:\""
+else
+  alias log_info="echo \"INFO:\""
+  alias log_error="echo \"ERROR:\""
+  alias log_warning="echo \"WARN:\""
+fi
+
+set -e
+
+# default to relative path if INSTALL_PATH is not set
+INSTALL_PATH=${INSTALL_PATH:-$(realpath ./.slsa-provenance)}
```

Comment:
For some reason my MacOS did not have a ``realpath`` command in ZSH or Bash(3.2). This was no issue on Ubuntu or my Windows machine. Not sure what is going on here.

Reply:
MacOS has an old version of Bash by default

```shell
brew update
brew install bash
```